### ISSUE TITLE
Test for AggregatedCalculation not passing yet

### ISSUE THREAD

**Issue by FBumann** [FBumann/flixOpt#52]
Aggregation itself gets parametrized correctly. Results match Test.
COntinue...

**Comment by FBumann** [comment]
# Comparing the cluster indices
Clustered indices shows, that cluster 2 and 3 are swapped. Might this be the problem?
## New
```
starts = {key: [arr[0] for arr in val] for key, val in self.get_cluster_indices().items()}
```
starts -> {0: [168, 264], 1: [24, 48, 72], 2: [144, 240], 3: [96, 192], 4: [0], 5: [120, 216]}

## Old
```
starts = {key: [arr[0] for arr in val] for key, val in self.getIndexVectorsOfClusters().items()}
```
starts -> {0: [168, 264], 1: [24, 48, 72], 2: [96, 192], 3: [144, 240], 4: [0], 5: [120, 216]}

**Comment by FBumann** [comment]
# SOLUTION
Due to an error in cAggregatedModeling (the old one), the Storage Flows are not fixed, even if .fixStorageFlows = True.

The Test is therefore corrected to NOT fix Storage Flows.

**Comment by FBumann** [comment]
Closed with #55
